samples/browser/src/main/java/org/mozilla/samples/browser/IntentReceiverActivity.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser
+
+import android.os.Bundle
+import org.mozilla.samples.browser.ext.components
+
+class IntentReceiverActivity : BrowserActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        components.sessionIntentProcessor.process(intent)
+        super.onCreate(savedInstanceState)
+    }
```

Comment:
Oh, interesting... the IntentReceiverActivity *is* a BrowserActivity instead of launching it? Won't this duplicate the browser if there are multiple non-custom-tab VIEW intents or if we launch the app  normally and then send a normal intent? Anyhow, at this point I would prefer merging this and investigating this later.